ui-admin/src/portal/siteContent/HtmlPageEditView.tsx

```diff
@@ -48,7 +52,10 @@ const HtmlPageView = ({ htmlPage, updatePage, readOnly }: HtmlPageViewProps) =>
             aria-label={'Insert a blank section'}
             tooltip={'Insert a blank section'}
             disabled={readOnly}
-            onClick={() => insertNewSection(index + 1, { id: '', sectionType: 'HERO_WITH_IMAGE' })}>
+            onClick={() => insertNewSection(index + 1, {
```

Comment:
I think this is the same markup as above-- you can pull this into a new renderAddSectionButton helper function (or component) that takes the index as an argument